File: Python Projects/additive_inverse.py
from operator import mod


element = int(input("enter element number: "))
modulus = int(input("enter modulus: "))


ai_set = []
index = 0
while (index<modulus):
    ai_set.append(index)
    index += 1

if (element not in ai_set):
    print("Invalid Element selection! Restart the program and Please select a number between 0 to " + str(ai_set[len(ai_set)-1]))
else:
    for number in ai_set:
        ai = number + element
        if mod(ai, modulus)== 0:
            print("Additive inverse of " + str(element) + " is " + str(number) + ".")
            break
            

File: Go-projects/additive_inverse.go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	fmt.Println("Enter the modulus number: ")
	var modulus int
	fmt.Scanln(&modulus)

	fmt.Println("Enter your element value: ")
	var element int
	fmt.Scanln(&element)

	index := 0
	var ai_set []int

	for index < modulus {
		ai_set = append(ai_set, index)
		index += 1
	}

	for number := range ai_set {
		ai := number + element
		if ai%modulus == 0 {
			fmt.Print("Additivie inverse of " + strconv.Itoa(element) + " is " + strconv.Itoa(number))
		}
	}

}
